app/views/proposals/new.html.haml

```diff
@@ -3,6 +3,14 @@
     .col-md-12
       .page-header
         %h1 New Proposal
+  - if @program.cfp_open?
+    - if @program.cfp.description.length>0
+      .row
+        .col-md-12
+          %p.lead
+            %b
+              Description:
```

Comment:
Do we actually need to show the word description? Why not just show the text?
Is it helpful if I see the following text, for example?

> Description: Note that you need to be at your booth at all times during the conference!

Reply:
Instead of description we could show something which catches user attention like important or so..

Reply:
That's indeed another good idea. I was thinking to show nothing at all, just the text itself. 
The text may be a description, it may be important information, it may be something else.

Reply:
Yeah, so could we keep it as "important information"?
Also could we move this "important info" out of the condition checking statement, because some info like days left is shown and won't be left empty anyways?

Reply:
I have no idea. I think we don't have to show anything.

If organizers want to show Important there, they can do so by adding it in the beginning of their description text. Markdown gives them tons of possibilities to format this to their liking. 

So let's not show anything except for the description itself and wrap this up ;)

Reply:
Cool! So I would fix the show page and condition statement.